resources/documentation/multiple_icebergs.rst

```diff
@@ -227,45 +291,226 @@ And add management2 to iceberg2, by editing /etc/bluebanquise/inventory/cluster/
 
-Then deploy repositories_server role, and nfs_client role, so that management2 can get repositories localy and distribute them for iceberg2:
+*/var/www/html/repositories* and */etc/bluebanquise* from mngt1 are now mounted
+on mngt2, and httpd server is running on mngt2.
+
+.. image:: images/multiple_icebergs_5.svg
+
+Now, mngt2 can be autonomous and do not need to be part of iceberg1.
+Deploy the whole configuration on it:
 
 .. code-block:: text
 
-  ansible-playbook /etc/bluebanquise/playbooks/management2.yml -t repositories_server,nfs_client --extra-vars j2_current_iceberg=iceberg1
+  ansible-playbook /etc/bluebanquise/playbooks/mngt2.yml
+
+And now mngt2 act as iceberg2 management, and can provide packages to its nodes.
 
-Now, management2 can be autonomous. Deploy the whole configuration on it:
+.. image:: images/multiple_icebergs_6.svg
+
+And proceed as usual to add more hosts into iceberg2 and deploy them,
+this time from mngt2.
+
+Redo this same process for each additional island.
+
+Interconnect and job scheduler (HPC only)
+-----------------------------------------
+
+Now that your icebergs are up and running comes the question of the interconnect
+(if exist) and the job scheduler (by default Slurm). Same question can be made
+for the storage: you may need all nodes to reach a network FS (Lustre, BeeGFS,
+etc.).
+
+.. image:: images/multiple_icebergs_7.svg
+
+For storage, it should be straightforward: once the storage is online and
+reachable over the interconnect, all nodes can mount it.
+
+The Slurm setup is a little bit more complex. You will need a unified network to
+allow your nodes to be able to reach the same Slurm controller, mostly running
+on the mngt1 server.
+
+But you will also need to ensure direct hostnames resolution of all computes
+nodes is done on the interconnect, and not on the internet. Why ? Simply because
+when parallel computations take places, Slurm will provide to the instance nodes
+hostnames as target, and so if nodes need to reach each other through ethernet,
+nodes from once iceberg will not be able to reach nodes from other icebergs, and
+so parallel computations will not initialize.
+
+Example: user job is asking for 100 nodes, the whole cluster is free, and each
+iceberg contains 80 nodes. Slurm will allocate 80 nodes from iceberg1, lets say
+c[001-080] and 20 nodes from iceberg2, c[081-100]. The final mpirun command
+will receive as hosts target c[001-100]. c001 will be able to communicate with
+c002, but not with c081, as iceberg ethernet networks are isolated. But if c081
+means c081 over the interconnect network, then since this network is unified,
+c001 will be able to reach c081 and initialize MPI run.
+
+.. note::
+  It is possible to set routing between icebergs over ethernet, but this is not
+  in the scope of this documentation.
+
+To achieve direct computes hosts resolution over interconnect, ensure the
+interconnect network interface is first in the network_interface list of each
```

Comment:
Should we use the concept of "preferred network" for the first network interface in the list?

Reply:
To achieve direct computes hosts resolution over interconnect, ensure the
interconnect network interface is first in the network_interface list of each
compute node, which is the preferred network. ?

Reply:
yes